modules/setting-up-posix-shared-memory.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * storage/understanding-ephemeral-storage.adoc
+
+[id="setting-up-posix-shared-memory_{context}"]
+= Setting up POSIX Shared Memory
+
+
+The following procedure shows how to set up POSIX shared memory between two containers.
+
+.Procedure
+
+1. Edit the pod YAML file to specify the tmps volume for the containers, as shown in the `shared-memory.yaml` file.
```

Comment:
Maybe this is my ignorance. What is a tmps volume? I don't see this term used in the docs. Do we need to explain what this is or what the acronym stands for?

Reply:
Do you need to specify what pod this is?

Reply:
It should be tmpfs; have changed it.